client/state/reader/follows/actions.js

```diff
@@ -47,3 +52,52 @@ export function recordUnfollow( url ) {
 		} );
 	};
 }
+
+/**
+ * Returns an action object to signal that followed sites have been received.
+ *
+ * @param  {Array}  follows Follows received
+ * @return {Object} 		Action object
+ */
+export function receiveFollows( follows ) {
```

Comment:
>  Like maybe that we sometimes want to receive outside of the normal api response

We do do that sometimes, especially for things like posts, which could come in from any of a number of api responses. It's not super common though. 

The idea is to separate receiving a thing from the machinery needed to gather the thing. Eventually I think all the request / success / failure stuff may just go away and be hidden in middleware.